server/app.js

```diff
@@ -1,17 +1,35 @@
 import express from 'express';
 import bodyParser from 'body-parser';
 import logger from 'morgan';
+import router from './routes/route';
 
+// Set up the express app
 const app = express();
 
+// Log requests to the console.
 app.use(logger('dev'));
+
+// Parse incoming requests data
 app.use(bodyParser.json());
 app.use(bodyParser.urlencoded({
   extended: true,
 }));
+app.use('/api/v1', router);
+
+// welcome route
+app.get('/', (req, res) => {
+  res.status(200).json({
+    status: '200',
+    message: 'Welcome to Politico!'
+  });
+});
 
-app.all('*', (req, res) => {
-  res.status(404).send('The page you are looking for was not found');
+// Default catch-all route that sends back an error message for wrong routes
+app.all('/*', (req, res) => {
+  res.status(404).send({
+    status: '404',
+    message: 'The page you are looking for was not found'
```

Comment:
The sentence should be `The page you were looking for was not found!`

Reply:
Alright, Noted and implemented